tron/core/actionrun.py

```diff
@@ -165,6 +167,12 @@ def __init__(
         self.output_path = output_path or filehandler.OutputPath()
         self.output_path.append(self.id)
         self.context = command_context.build_context(self, parent_context)
+        self.retries = retries
```

Comment:
this is really minor, but it took me reading the code to figure out whether this variable meant 'number of retries already attempted' or 'number of retries remaining'. could you instead rename this 'remaining_attempts', or seperate into ``self.current_attempt`` and ``self.max_attempts``?